internal/commands/package_buildpack.go

```diff
@@ -71,11 +68,27 @@ func PackageBuildpack(logger logging.Logger, client BuildpackPackager, packageCo
 		}),
 	}
 	cmd.Flags().StringVarP(&flags.PackageTomlPath, "package-config", "p", "", "Path to package TOML config (required)")
-	cmd.MarkFlagRequired("package-config")
+	cmd.Flags().StringVarP(&flags.PackageTomlPath, "config", "c", "", "Path to package TOML config (required)")
+
+	// TODO: Mark config required and remove package-config after release of pack v0.12
```

Comment:
Our practice is not to leave `TODO` without an issue associated with it. Please create an issue and provide that issue number in this comment so that we make sure we follow-up.